packages/tags/src/_remove.css

```diff
@@ -77,6 +75,18 @@
   display: none;
 }
 
+.c-tag.c-tag--blue .c-tag__remove,
+.c-tag.c-tag--green .c-tag__remove,
+.c-tag.c-tag--grey .c-tag__remove,
+.c-tag.c-tag--kale .c-tag__remove,
+.c-tag.c-tag--red .c-tag__remove {
```

Comment:
Not sure if this is better but we could use `:matches()` here e.g.

```css
.c-tag:matches(
  .c-tag--blue, 
  .c-tag--green, 
  .c-tag--grey, 
  .c-tag--kale, 
  .c-tag--red
) :matches(.c-tag__remove) {
  background-image: var(--zd-tag--color__remove-background-image);
} 
```

Reply:
Huh. TIL and postcss results in the original output.